File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem74_DailyTemperaturesTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;

class Problem74_DailyTemperaturesTest {

    private final Problem74_DailyTemperatures test = new Problem74_DailyTemperatures();

    @Test
    void dailyTemperatures() {
        int[] temperatures = new int[]{73, 74, 75, 71, 69, 72, 76, 73};
        int[] expected = new int[]{1, 1, 4, 2, 1, 1, 0, 0};

        assertArrayEquals(expected, test.dailyTemperatures(temperatures));
    }
}

File: JavaScript/leetcode75/src/test/Problem74_DailyTemperatures.test.js
const dailyTemperatures = require('../main/Problem74_DailyTemperatures');

test('should return difference', () => {
  const temperatures = [73, 74, 75, 71, 69, 72, 76, 73];
  const result = [1, 1, 4, 2, 1, 1, 0, 0];

  expect(dailyTemperatures(temperatures)).toStrictEqual(result);
});
